### ISSUE TITLE
Bug: Selecting Range date shows a console error

### ISSUE THREAD

**Issue by kfirba** [mathieustan/vue-datepicker#13]
Hey.

When selecting a date range (even on the demo site), I get the following error:

```
TypeError: Cannot convert undefined or null to object
    at Function.values (<anonymous>)
    at a.isRangeSelected (vue-datepicker.esm.js:3610)
    at nr.get (vue.runtime.esm.js:4473)
    at nr.evaluate (vue.runtime.esm.js:4578)
    at a.isRangeSelected (vue.runtime.esm.js:4830)
    at a.handleMouseMove (vue-datepicker.esm.js:3882)
    at ne (vue.runtime.esm.js:1854)
    at HTMLDivElement.n (vue.runtime.esm.js:2179)
    at HTMLDivElement.Qo.i._wrapper (vue.runtime.esm.js:6911)
```

This stack error is referring to the `DatePickerAgenda.vue` when we attempt to calculate the `isRangeSelected` computed property:

```js
isRangeSelected () {
  if (!this.range) return false;
  return Object.values(this.mutableDate).every(date => Boolean(date)); // Error here (Object.values) is called on "undefined"
},
```

This error does not affect the functionality as the datepicker still works, but it throws many errors in the console.

You can reproduce the issue on the demo website at the `Range (with presets)` section:

1. Open the datepicker and choose a starting and ending date and confirm your selection. Sometimes the error occurs during this step. Sometimes you have to do the next step:
2. open the datepicker again, now press "Cancel". It sometimes takes a few time to open and close the datepicker in order to make it throw an error.

**Comment by mathieustan** [comment]
Hello. 

I tried many time on demo site, and I can't see this error. 
I'll take a look at code to see if there is anything weird

**Comment by kfirba** [comment]
@mathieustan Try to play with the range datepicker. Open it, close it, select dates and confirm, select dates and cancel, rinse and repeat until you see the error. I can assure you it's there :/

**Comment by cristiancastrodc** [comment]
I've been able to reproduce the bug... After a lot trying. I sent you a record of how to reproduce it.

Also, I tried again repeating the steps and the error showed before, just when I finished the third selection.

[vue-datepicker - Mozilla Firefox 2019-09-18 14-54-48.zip](https://github.com/mathieustan/vue-datepicker/files/3628117/vue-datepicker.-.Mozilla.Firefox.2019-09-18.14-54-48.zip)

**Comment by mathieustan** [comment]
Should be fixed in latest release [0.1.9](https://github.com/mathieustan/vue-datepicker/releases/tag/0.1.9)
